gravitee-io/gravitee-access-management
# feat: conditional policies on flow steps

closes: https://github.com/gravitee-io/issues/issues/7016

## How to Test

1. Create a Policy flow (either at domain or application level) to allow a validation of Conditional policy (see below)

![Screenshot 2022-03-08 at 22 05 07](https://user-images.githubusercontent.com/8531515/157337219-b8b1e751-088f-4f26-b61b-5043d646a1dd.png)
![Screenshot 2022-03-08 at 22 05 15](https://user-images.githubusercontent.com/8531515/157337227-a0433440-ee56-417e-a9a6-0ab0565e2691.png)
![Screenshot 2022-03-08 at 22 05 28](https://user-images.githubusercontent.com/8531515/157337281-7993ac51-1deb-4c30-9801-1164551039db.png)

Reverse the condition in the condition of the step to be able to assert the use
